L1Trigger/L1TGlobal/src/CorrCondition.cc

```diff
@@ -370,13 +378,15 @@ const bool l1t::CorrCondition::evaluateCondition(const int bxEval) const {
         phiIndex0 = (candMuVec->at(cond0bx, obj0Index))->hwPhiAtVtx();  //(*candMuVec)[obj0Index]->phiIndex();
         etaIndex0 = (candMuVec->at(cond0bx, obj0Index))->hwEtaAtVtx();
         etIndex0 = (candMuVec->at(cond0bx, obj0Index))->hwPt();
+	uptIndex0 = (candMuVec->at(cond0bx, obj0Index))->hwPtUnconstrained(); // Added by R.Cavanaugh for displaced muons
```

Comment:
Please remove meaningless comments.  Git versioning system keeps track of authors of commits.

Reply:
@rekovic 
Many thanks for the helpful request to remove comments.  Done.